tests/unit/classifier/LDA_unittest.cc

```diff
@@ -176,3 +206,35 @@ TEST_F(LDATest, DISABLED_CheckProjection_SVD)
 	EXPECT_NEAR(+6.90668279, projection_SVD[8], epsilon);
 	EXPECT_NEAR(+7.96084156, projection_SVD[9], epsilon);
 }
+
+//FLD template testing
+TEST(LDA, FLD_template_test_float32)
```

Comment:
Hmm, I don't understand what the macro should do - should it compact the below into a single line?

```
TEST(LDA, FLD_template_test_floatmax)
{
	check_eigenvectors_fld<floatmax_t>();
}
```